web/src/engine/websites/templates/KeyoApp.ts

```diff
@@ -1,6 +1,9 @@
 //KeyoApp template : websites Similar to MagusManga, StarBoundScan, LuaScans. One page Manga list & hosting images on cdn.igniscans.com
 export const pagesScript = `
-	new Promise ( resolve => resolve( [...document.querySelectorAll('.myImage')].map(image => 'https://cdn.igniscans.com/uploads/'+ image.getAttribute('uid'))));
+	new Promise ( resolve => {
+		const realUrl = document.documentElement.innerHTML.match(/realUrl\\s*=\\s*\`([^\`]+?)\\$/)[1];
```

Comment:
While you were writing this i was fixing the tests problem i changed my solution. Its the same as yours except the "count". Since the meaning of count is "how much time did i try to fetch the image" its irrelevant to us and fixed at 1.